staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -119,6 +121,12 @@ type GenericAPIServer struct {
 	swaggerConfig *swagger.Config
 	openAPIConfig *openapicommon.Config
 
+	// Expose the registered OpenAPI Services and built static OpenAPI spec
+	// if openAPIConfig is non-nil
+	OpenAPIService          *handler.OpenAPIService
```

Comment:
add comments that the former is about /swagger.json and the latter is /openapi/v2

Reply:
done